.travis.yml

```diff
@@ -0,0 +1,30 @@
+language: cpp
+compiler: gcc
+sudo: require
+dist: trusty
+
+script:
+  - ./autogen.sh # Undocumented
+  - ./configure --prefix=/usr
+  - make -j$(nproc)
```

Comment:
I would feel a bit more comfortable if this were also followed by `make check` to run the tests. The tests by default only add perl and gnu diff as an additional dependency, but that is the main diff in Linux so it's pretty much always there. You need perl for fix-qdf anyway.

Reply:
Sure, added.